zerver/migrations/0402_alter_realm_org_type.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.0.6 on 2022-07-28 19:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0401_migrate_old_realm_reactivation_links"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="realm",
+            name="org_type",
+            field=models.PositiveSmallIntegerField(
+                choices=[
+                    (10, "Business"),
+                    (20, "Open-source project"),
+                    (30, "Education (non-profit)"),
+                    (35, "Education (for-profit)"),
+                    (40, "Research"),
+                    (50, "Event or conference"),
+                    (60, "Non-profit (registered)"),
+                    (70, "Government"),
+                    (80, "Political group"),
+                    (90, "Community"),
+                    (100, "Personal"),
+                    (1000, "Other"),
+                    (0, "Unspecified"),
+                ],
+                default=0,
+            ),
+        ),
+    ]
```

Comment:
Since changing the order is a Django-only change (not visible to the database), one can avoid creating a new migration by just changing the existing migrations that define the `org_type` field to have the revised format.

Reply:
Like this:

```
diff --git a/corporate/migrations/0012_zulipsponsorshiprequest.py b/corporate/migrations/0012_zulipsponsorshiprequest.py
index 4402e1d2fa..e054ce5dd7 100644
--- a/corporate/migrations/0012_zulipsponsorshiprequest.py
+++ b/corporate/migrations/0012_zulipsponsorshiprequest.py
@@ -27,7 +27,6 @@ class Migration(migrations.Migration):
                     "org_type",
                     models.PositiveSmallIntegerField(
                         choices=[
-                            (0, "Unspecified"),
                             (10, "Business"),
                             (20, "Open-source project"),
                             (30, "Education (non-profit)"),
@@ -40,6 +39,7 @@ class Migration(migrations.Migration):
                             (90, "Community"),
                             (100, "Personal"),
                             (1000, "Other"),
+                            (0, "Unspecified"),
                         ],
                         default=0,
                     ),
diff --git a/zerver/migrations/0333_alter_realm_org_type.py b/zerver/migrations/0333_alter_realm_org_type.py
index df6f29e692..dc95cce052 100644
--- a/zerver/migrations/0333_alter_realm_org_type.py
+++ b/zerver/migrations/0333_alter_realm_org_type.py
@@ -23,7 +23,6 @@ class Migration(migrations.Migration):
             name="org_type",
             field=models.PositiveSmallIntegerField(
                 choices=[
-                    (0, "Unspecified"),
                     (10, "Business"),
                     (20, "Open-source project"),
                     (30, "Education (non-profit)"),
@@ -36,6 +35,7 @@ class Migration(migrations.Migration):
                     (90, "Community"),
                     (100, "Personal"),
                     (1000, "Other"),
+                    (0, "Unspecified"),
                 ],
                 default=0,
             ),
```

Can you also add something to the commit message to explain why this is being moved? It's a dictionary, so it seems like this reordering should be effectively a noop.

Reply:
That said, we should instead just skip this commit, and instead just not show organizations with `unspecified` except in the All list, if that's easy. (Also fine would be to just not show them at all and we can periodically manually deal with it; it's a legacy situation not likely to come up much)

Reply:
Okay, removed `unspecified` and kept them under `All`.